.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb-base",
```

Comment:
I'm not familiar with what airbnb defines as their base for linting. Could you elaborate please.